sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -995,7 +996,7 @@ def test_zero_estimator_reg():
     est.fit(X_reg, y_reg)
     y_pred = est.predict(X_reg)
     mse = mean_squared_error(y_reg, y_pred)
-    assert_almost_equal(mse, 0.52, decimal=2)
+    assert_almost_equal(mse, 0.13, decimal=2)
```

Comment:
Instead of comparing to a hardcoded value, maybe we could just compare that `est` has a better MSE than a simple baseline such as `DummyRegressor(strategy="mean")` that constantly predict the mean `y_reg`?